test/utils/runners.go

```diff
@@ -1588,10 +1589,23 @@ func attachSecrets(template *v1.PodTemplateSpec, secretNames []string) {
 			Name:      name,
 			MountPath: fmt.Sprintf("/%v", name),
 		})
+		envFrom = append(envFrom, v1.EnvFromSource{
+			Prefix: name,
+			SecretRef: &v1.SecretEnvSource{
+				LocalObjectReference: v1.LocalObjectReference{
+					Name: name,
+				},
+			},
+		})
+		imagePullSecrets = append(imagePullSecrets, v1.LocalObjectReference{
+			Name: name,
+		})
 	}
 
+	template.Spec.ImagePullSecrets = imagePullSecrets
 	template.Spec.Volumes = volumes
 	template.Spec.Containers[0].VolumeMounts = mounts
+	template.Spec.Containers[0].EnvFrom = envFrom
```

Comment:
Can we also update this test to validate that these values are set correctly in the container?